mcfix_base/models/res_currency.py

```diff
@@ -0,0 +1,11 @@
+from odoo import api, models
+
+
+class Currency(models.Model):
+    _inherit = "res.currency"
+
+    @api.multi
+    def amount_to_text(self, amount):
```

Comment:
I am not sure why you introduce this function as part of multi-company fix: maybe you could add a comment or docstring

Reply:
Provided details